go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/containerd/containerd
 
-go 1.19
+go 1.21
+
+toolchain go1.21.7
```

Comment:
If the min version update is required, then this will require an update to BUILDING.md as well.

Alternatively, we could consider having the `make vendor` target do `go mod tidy -compat=1.19` so the toolchain isn't pushing these min version updates on users unless absolutely required.

Reply:
Oh thank you I didn't realize the `-compat` flag was there ❤️.
I'll update this after I get the hcsshim changes in mentioned by @kiashok

Reply:
This is no longer being changed, yay.